NEWS.md

```diff
@@ -1,5 +1,8 @@
 # riskassessment (development version)
 
+### Enhancements
+*Added introjs for file browser & function explorer( #581)
```

Comment:
```suggestion
*Added introjs for file browser & function explorer( #581)
```
We only use the sub-headers when we cut a release. We just list the NEWS as we go until then.

Reply:
updated